frontend/src/components/BackButton.jsx

```diff
@@ -0,0 +1,30 @@
+import { Link } from "react-router-dom";
+
+const BackButton = () => {
+  const containerStyle = {
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+  };
+
+  const buttonStyle = {
+    padding: "10px 20px",
+    fontSize: "16px",
+    borderRadius: "5px",
+    backgroundColor: "#0d37de",
```

Comment:
This background colour does not match the application theme. **We should change it.** 

On Windows and MacOS, you can get extensions/applications that can get colour hexcodes from anywhere in your screen - checkout PowerToys for Windows.